stylesheet.css

```diff
@@ -0,0 +1,177 @@
+body {
+  background-color: #e5e5e5;
+  background-image: url("image/Topcircles.png");
+  font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif, "Roboto";
+  margin: auto;
+}
+
+.outer-maincont {
+  display: flex;
+  flex-direction: column;
+  margin: auto;
+  background-color: #191765;
+  max-width: 375px;
+  min-height: 100vh;
+  padding-bottom: 10px;
+  color: white;
+}
```

Comment:
- You did a great job by making your page responsive on the screen of 375px but it make some modifications in your navigation bar and the footer as it is shown in the picture below kind fix that +1: 
  - for the nav bar, I can suggest you remove the display flex  in the `lower-bar `so that the space between many and the logo will be applicable :-1:  
   - for the icons in the footer which are not visible try to remove the fixed width you give to some sections and to the all-screen so that it will make your page correctly displayed
## navigation bar
![image](https://user-images.githubusercontent.com/95297251/198712941-b2266e24-97c7-4dca-b734-e41835cdb896.png)
## footer 
![image](https://user-images.githubusercontent.com/95297251/198714412-427de051-7038-429e-9356-c58f2f6b3dfd.png)